addons/web/static/src/legacy/js/core/dom.js

```diff
@@ -545,11 +545,14 @@ var dom = {
      * @param {number} [options.forcedOffset]
      *      offset used instead of the automatic one (extraOffset will be
      *      ignored too)
+     * @param {JQuery} [options.$scrollable] the $element to scroll
      * @return {Promise}
      */
     scrollTo(el, options = {}) {
         const $el = $(el);
-        const $scrollable = $el.parent().closestScrollable();
+        const $scrollable = options.$scrollable || $el.parent().closestScrollable();
+        const isInOneDocument = $scrollable[0].ownerDocument === $el[0].ownerDocument;
+        const $iframe = !isInOneDocument && $scrollable.find('iframe').filter((i, node) => $(node).contents().has($el));
```

Comment:
Those lines probably could gain some explanations ^^ don't understand them. I suppose it is quite clear for you, but for external people, well :D ...

Reply:
Haha sure ok :p

```js
// If $scrollable and $el are not in the same document, we can safely
// assume $el is in an $iframe. We retrieve it by filtering the list of
// iframes in $scrollable to keep only the one that contains $el.
```